libr/bin/format/elf/elf.h

```diff
@@ -80,19 +99,19 @@ typedef struct r_bin_elf_lib_t {
 struct Elf_(r_bin_elf_obj_t) {
 	Elf_(Ehdr) ehdr;
 	Elf_(Phdr)* phdr;
-	Elf_(Shdr)* shdr;
+	Elf_(Shdr) * shdr;
```

Comment:
either keep it as it was, or fix it with `Elf_(Shdr) *shdr;`

Reply:
How this seems to be the fault of `./sys/pre-commit-indent.sh`

Reply:
yeah... unfortunately it does not work as it should :(